adb_graphics/figures/maps.py

```diff
@@ -91,10 +95,11 @@ def __init__(self, airport_fn, ax, **kwargs):
         self.ax = ax
         self.grid_info = kwargs.get('grid_info', {})
         self.model = kwargs.get('model')
+        self.plot_airports = kwargs.get('plot_airports', True)
```

Comment:
plot_airports has been removed from the __init__ args, and retrieved using kwargs.get with a default of True.